osdep/timer-win32.c

```diff
@@ -27,28 +29,36 @@
 
 static LARGE_INTEGER perf_freq;
 
-// ms values
-static int hires_max = 50;
-static int hires_res = 1;
+static int64_t hires_max = MP_TIME_US_TO_NS(50000);
+static int64_t hires_res = MP_TIME_US_TO_NS(1000);
 
-int mp_start_hires_timers(int wait_ms)
+// NtSetTimerResolution allows setting the timer resolution to less than 1 ms.
+// Resolutions are specified in 100-ns units.
+// If Set is TRUE, set the RequestedResolution. Otherwise, return to the previous resolution.
+NTSTATUS NTAPI NtSetTimerResolution(ULONG RequestedResolution, BOOLEAN Set, PULONG ActualResolution);
+// Acquire the valid timer resolution range.
+NTSTATUS NTAPI NtQueryTimerResolution(PULONG MinimumResolution, PULONG MaximumResolution, PULONG ActualResolution);
```

Comment:
I think you can remove `NtQueryTimerResolution` it should be declared in headers.